types/src/ledger_info.rs

```diff
@@ -28,10 +28,23 @@ use std::{
     ops::{Deref, DerefMut},
     sync::{
         atomic::{AtomicBool, Ordering},
-        Arc,
+        Arc, OnceLock,
     },
 };
 
+/// Global waypoint version storage for bypassing verification of historical data
+static WAYPOINT_VERSION: OnceLock<Version> = OnceLock::new();
+
+/// Initialize the waypoint version (should be called once during node startup)
+pub fn set_waypoint_version(version: Version) {
```

Comment:
This function isn't used anywhere. So, the waypoint version is set.

Reply:
good catch. I acidentally removed the set. added it back